### DISCUSSION TITLE
store.findChatByJid only works the first time

### DISCUSSION THREAD

**Discussion by luckebar** [Auties00/WhatsappWeb4j#461] [Q&A]
store.findChatByJid only works the first time the second time rest is called and I use reconnect() the store.findChatByJid method does not return any chat

Do I need to release the store or chat somehow?

Thank you.

**Comment by codespearhead** [comment]
Can you provide us with a [reproducible](https://en.wikipedia.org/wiki/Minimal_reproducible_example)?

**Comment by luckebar** [comment]
I solved it using a custom serializer where with ProtobufControllerSerializer I saved in a folder named with the from number and then I always resume the last chat of this folder. This is how it works for me. This is because I have many from numbers
